### ISSUE TITLE
Dismiss card view via swipe does not trigger `.cancel` callback

### ISSUE THREAD

**Issue by tombarton** [onfido/onfido-ios-sdk#122]
### What was the expected behaviour?

As of iOS 13, the Onfido SDK follows standard Apple design and uses a card-style view. This functionality allows the user to click on a call to action (CTA) to dismiss the view or to swipe down instead. I would expect both clicking the CTA and swiping down would dismiss the view and fire the `.cancel` callback.

### What happened instead?

Swiping down dismisses the view but does **not** fire the `.cancel` callback.

### Version info:

- Cocoapods (if applicable): 1.8.4
- Onfido SDK (Debug/Release): 16.0.0
- iOS: 13.1.0
- Xcode: 11.1
- Device/Simulator: iPhone 11
- Device/Simulator language: English

### Steps to reproduce:

1. Load SDK into a view by following the instructions in the README.md

```self.present(onfidoRun, animated: true, completion: nil)```

2. Swipe view down to dismiss .

**Comment by anuragajwani** [comment]
Hi @tombarton,

Thanks for raising the issue. The only two presentation styles that we support are `.formSheet` for iPads and `.fullScreen` for iPads and iPhones.

We haven't really needed to specify until iOS 13 about this as prior to iOS 13 the default was full screen presentation style. We have created a ticket to update the `README.md` file to specify this clearly.

As a guide we always set the presentation style on our SampleApp contained within this repo. Checkout https://github.com/onfido/onfido-ios-sdk/blob/master/SampleApp/SampleApp/ViewController.swift#L70.

Sorry for the inconvenience caused. We'll update this issue once the `README.md` has been updated.

Anurag
